src/Altinn.Notifications/appsettings.json

```diff
@@ -13,7 +13,7 @@
     "EnableDBConnection": true
   },
   "NotificationConfig": {
-    "DefaultEmailFromAddress": "noreply@altinn.no",
+    "DefaultEmailFromAddress": "noreply@altinn.cloud",
```

Comment:
This might cause issues in tt02 and production. Make sure we add correct values for those environments in the values file.

Reply:
the `values.yaml` file has the following value:
`noreply@altinn.no` for the environments tt02 and prod.
`noreply@altinn.cloud` for the environments development, at21, at22, at23, at24 and yt01.